### PR TITLE
feat: loading screen polishing

### PR DESCRIPTION
**Pull request by lorux0**
Introduce visual improvements on the loading screens:
- background blur
- transition animations
- layout
- randomization fixes
- tip rotation duration

Also introduces the loading screens when teleporting to any parcel, either from the debug panel or the minimap.

# How to test

1. Start the initial flow as expected
2. Change realm from the debug panel
3. Teleport to different spawn points or coordinates
4. Rotate tips
5. Check that the loading screen is always displayed as expected
6. Open the minimap and click to jumpin to any parcel

### CODE REVIEW

**Review by mikhail-dcl — CHANGES_REQUESTED** [review]
Hey Nico, it will not work this way: you should respect empty scenes just like it's respected in the single-parcel version.
- You should modify ByParcelAsync(Vector2Int parcel, CancellationToken ct) to pass a readonly list instead. The endpoint already supports the list, currently it's requested with one parcel only
- You should filter out all empty parcels and create reports for non-empty ones. Otherwise, empty parcels will be never fullfilled

**Review by anicalbano — APPROVED** [review]
✔️ Reviewed and approved by QA

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs [hunk 1]
```diff
@@ -65,30 +82,82 @@ public void AddTip(SceneTips.Tip tip)
...
+                                        .AttachExternalCancellation(ct);
+
+            await UniTask.WhenAll(tipFadeTask, backgroundFadeTask);
+        }
+
+        public async UniTask HideTipWithFadeAsync(int index, float duration, CancellationToken ct)
+        {
+            UniTask tipFadeTask = tips[index]
+                                 .RootCanvasGroup.DOFade(0f, duration)
+                                 .AsyncWaitForCompletion()
```
Comment (mikhail-dcl) [inline Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs:148]: You should not create an intermediate task and then convert it to `UniTask`. It creates an overhead: just use `DOTweenAsyncExtensions` directly (we already have a couple of use cases). You just need a `UniTask.DOTween` assembly for that.
Please consider changing all the occurences.

File: Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs [hunk 2]
```diff
@@ -65,30 +82,82 @@ public void AddTip(SceneTips.Tip tip)
...
+                                 .AsUniTask();
+
+            UniTask backgroundFadeTask = backgrounds[index]
+                                        .RootCanvasGroup.DOFade(0f, duration)
+                                        .AsyncWaitForCompletion()
+                                        .AsUniTask()
+                                        .AttachExternalCancellation(ct);
```
Comment (mikhail-dcl) [inline Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs:155]: These extensions also have a built-in `CancellationToken` support so you don't need to work it around with `AttachExternalCancellation`. 
`AttachExternalCancellation` is general is a bad practive and should be used for code out of our control only: it does not clean-up any resources under the hood: it stops spininng the wrapped task only that may lead to resources leak.

**Review by mikhail-dcl — CHANGES_REQUESTED** [review]

**Comment by mikhail-dcl** [comment]
Additionaly: are you sure that design-wise we should use `Teleport To Parcel` for Navmap instead of `Teleport To Spawn Point`?

**Comment by anicalbano** [comment]
Re checked 🟢

**Comment by lorux0** [comment]
> Additionaly: are you sure that design-wise we should use `Teleport To Parcel` for Navmap instead of `Teleport To Spawn Point`?

Changed to teleport to spawn point (validated with product)

**Review by mikhail-dcl — APPROVED** [review]

**Pull request merged**